pages/main/style.css

```diff
@@ -254,7 +254,7 @@ body{
 }
```

Comment:
It seems like you are using a different font than it's given in the design of Figma, for the logo, section titles/content, and footer contact entries. For all these entries 'Georgia' font can be used as well.

Your Design | Figma Design
---------------- | --------------
![shoirah](https://user-images.githubusercontent.com/79588769/166154142-2f1b1821-0d8a-42fa-9ac8-f6d87d89dad2.png)| ![cozyh](https://user-images.githubusercontent.com/79588769/166154176-7d7d05b3-2106-447a-a321-711e010a4dfd.png)
![shoiraho](https://user-images.githubusercontent.com/79588769/166154157-080d230e-1598-4311-a0a7-3dbd83ec92bf.png) | ![cozyho](https://user-images.githubusercontent.com/79588769/166154179-dd7beb78-93ec-4bdc-b871-294d81c10d0e.png)